src/models/operation-trait.ts

```diff
@@ -1,11 +1,10 @@
 import type { BaseModel } from './base';
 import type { BindingsMixinInterface, DescriptionMixinInterface, ExtensionsMixinInterface, ExternalDocumentationMixinInterface, TagsMixinInterface } from './mixins';
-import { SecurityRequirements } from './v2/security-requirements';
+import { SecurityRequirements } from './security-requirements';
 
 export interface OperationTraitInterface extends BaseModel, BindingsMixinInterface, DescriptionMixinInterface, ExtensionsMixinInterface, ExternalDocumentationMixinInterface, TagsMixinInterface {
-  id(): string;
-  hasOperationId(): boolean;
-  operationId(): string | undefined;
+  id(): string | undefined;
+  hasId(): boolean;
```

Comment:
I changed this because I think it's confusing. An operation shouldn't have an `id` and an `operationId`. The `operationId` is the `id` of the operation 😅 So yeah, changed it and changed tests too.

Reply:
Makes sense 👍 . Would you mind adding such a change into https://github.com/asyncapi/parser-api/pull/71 ?

Reply:
I don't think I understand 😅 Just to clarify, in v3 `operationId` is not mandatory, it just doesn't exist. The id comes from the key in the `operations` object.

Reply:
we have also `components.operations`, therefore `id` and `operationId` exist together.

Reply:
Yeah, something to check in this case with v2.

Reply:
For sure, what doesn't make sense is to have `operation.id` and `operation.operationId`. They're the same thing with different names. BTW, the same will happen with `messageId` eventually.

Reply:
from v2 spec point of view it makes sense and that's the problem 😄

Reply:
I don't think it does, even from v2 point of view. `operationId` was there to mean the `id` of the operation. If it's present, it should be the operation identifier. If it's not, ok, we can generate one if you want, but it was never meant to be an "extra" id.

Reply:
Hmm 🤔 I have a problem in general because in the v3 version people will be able to define operations in components and how to distinguish this identifier in `components.operations[x]` from `operationId` - that's why we have these two methods.`id()` method is more of a method used to filter operations by identifier (used as key in the `components.operations[x]` and `operations[x]`), but... we'll see how to figure it out and `id()` should return `operationId`. Can we fix it in next PRs? We need also to fix that in `parser-api` and remove `operationId()` method.

To clarify: I wrote about `components.operations[x]` and `operations[x]`, because in `parser-api` we have method `allOperations()` which should return all operations defined in given AsyncAPI document.

Reply:
I realized no one updated the parser-api with any change regarding this.

Reply:
All is fixed and there is no need to add anything else.